template.yml

```diff
@@ -39,7 +39,7 @@ Globals:
       Variables:
         defaultPowerValues: !Join [ ",", !Ref PowerValues ]
         minRAM: '128'
-        minCost: '0.000000208'
+        minCosts: 'ap-east-1=0.0000002865,af-south-1=0.0000002763,me-south-1=0.0000002583,default=0.000000208'
```

Comment:
Also, can you please integrate #76 here? The default base price should be `0.0000002083`.

Reply:
Will do.

Reply:
Sorted.